packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.types.ts

```diff
@@ -85,6 +85,11 @@ export interface IHoverCardProps extends React.HTMLAttributes<HTMLDivElement | H
    * Should block hover card or not
    */
   shouldBlockHoverCard?: () => void;
+
+  /**
+   * Set first focus into hover card, default should be true
+   */
+  setInitialFocus?: boolean;
```

Comment:
is the default true or false? If false, then the docs appear wrong, if true, we should flip the logic to have a prop like "disableInitialFocus" so that the exception state is the 'true' Boolean.

Reply:
@chaofeng2014 add `@default false` and update the description